### PR TITLE
update doc: installation supports brew

### PR DESCRIPTION
**Pull request by youngsterxyf**
R.T.

### CODE REVIEW

**Comment by youngsterxyf** [comment]
@astaxie is it ok?

File: README.md [hunk 1]
```diff
@@ -29,6 +29,16 @@ export PATH=$PATH:<your_main_gopath>/bin
 go get -u github.com/beego/bee
 ```
 
+------
+
+On **Mac OS X**, bee can be installed via [Homebrew](http://brew.sh/):
+
+```bash
+brew tap youngsterxyf/bee
```
Comment (astaxie) [inline README.md]: is that possible use `brew tap beego/bee`

**Comment by sergeylanzman** [comment]
Maybe add to brew repository beego
And install with 
```
brew instal bee // OR beego
```

**Comment by astaxie** [comment]
beego is a framework. while bee is a tool. but the `brew install bee` is already token by other app. But I think the homebrew script need to change to download from here https://github.com/beego/bee/releases/tag/v1.5.0

**Comment by astaxie** [comment]
I will update the brew later

**Pull request closed without merging**